src/rust/ide/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -508,7 +508,7 @@ impl Area {
                 let indent  = " ".repeat(4*builder.depth);
                 let skipped = if not_a_port { "(skip)" } else { "" };
                 println!("{}[{},{}] {} {:?} (tp: {:?}) (id: {:?})",indent,node.payload.index,
-                         node.payload.length,skipped,node.kind.variant_name(),node.tp(),node.ast_id);
+                    node.payload.length,skipped,node.kind.variant_name(),node.tp(),node.ast_id);
```

Comment:
And another debug print.

Reply:
This is behind the DEBUG flag here. When the flag is enabled, the ports are drawn in different way and this print is executed. This could arguably be done by logger, but then we will have much more cluttered output in the console, thus I left it for now.